index.html

```diff
@@ -635,15 +635,11 @@ <h3>Information Model</h3>
             <p>
                 [[[#directory-class-diagram]]] shows the information model of the
                 <a>Thing Description Directory</a>.
-                A directory can contain zero or more <a>TD</a>s.
-                For every TD, the directory may maintain the registration information for
-                bookkeeping and search purposes. The information may also include the
-                unique identifier of the TD within the directory.
-                This is useful for when a TD has been registered under an identifier
-                which differs from TD's internal `id` field.
-                When available, consumers may request the registration information
-                via the respective API.
-                A TD which embeds the registration information is called an <a>Enriched TD</a>.
+                A directory can contain zero or more <a>TD</a>s. For every TD, the directory may maintain the registration information for
+                bookkeeping and search purposes.
+		When available, consumers may request the registration information via the respective API.
+		A TD which embeds the registration information is called an <a>Enriched TD</a>. 
+		The registration information that can be specified for a TD is defined in the tdd ontology.
```

Comment:
Same issue as above. Is "tdd ontology" defined? Or this is the same as <a>TDD</a> ontology?

Reply:
Yes, finally I defined it as <a>Thing Description Directory</a> ontology